src/Microsoft.Azure.SignalR.Common/Endpoints/ServiceEndpoint.cs

```diff
@@ -26,6 +21,16 @@ public class ServiceEndpoint
 
         internal int? Port { get; }
 
+        /// <summary>
+        /// When there are server connections connected to the target endpoint for current hub, the endpoint is considered as Online; otherwise, Offline.
+        /// </summary>
+        public bool Online { get; internal set; } = true;
+
+        /// <summary>
+        /// When the target endpoint has client connections of current hub, the endpoint is considered as Active; otherwise, inactive.
+        /// </summary>
+        public bool IsActive { get; internal set; } = true;
```

Comment:
Can we call it something like `HasClientConnected`?

Reply:
From server's perspective, I think `IsActive` sounds acceptable though, with comments as further explanation.

Reply:
Add more comments